### PR TITLE
Update Communication examples

### PR DESCRIPTION
**Pull request by florianschoffke**

### CODE REVIEW

File: Standalone-Examples/E-Rezept-Workflow(gematik)/1.2.0/e72fee9b-75c7-4292-8ffc-cd81c151df8f.xml [hunk 1]
```diff
@@ -0,0 +1 @@
...4567890" /></identifier></recipient><sender><identifier><system value="https://gematik.de/fhir/sid/telematik-id" /><value value="3-SMC-B-Testkarte-883110000123465" /></identifier></sender><payload><extension url="https://gematik.de/fhir/erp/StructureDefinition/GEM_ERP_EX_AvailabilityState"><valueCoding><system value="https://gematik.de/fhir/erp/CodeSystem/GEM_ERP_CS_AvailabilityStatus" /><code value="20" /></valueCoding></extension><extension url="https://gematik.de/fhir/erp/StructureDefinition/GEM_ERP_EX_SupplyOptionsType"><extension url="onPremise"><valueBoolean value="true" /></extension><extension url="shipment"><valueBoolean value="false" /></extension><extension url="delivery"><valueBoolean value="true" /></extension></extension><contentString value="{&quot;version&quot;: 2,&quot;supplyOptionsType&quot;: &quot;onPremise&quot;,&quot;info_text&quot;: &quot;Ihre Bestellung steht zur Abholung bereit.&quot;,&quot;url&quot;: &quot;https://meine-apotheke.de/pickup/a9afa068&quot;,&quot;pickUpCodeHR&quot;: &quot;a9afa068&quot;,&quot;pickUpCodeDMC&quot;: {  &quot;pickUpCode&quot;: &quot;a9afa068&quot;,  &quot;interpretation&quot;: &quot;data-matrix&quot;}}" /></payload></Communication>
```
Comment (dennishahn) [inline Standalone-Examples/E-Rezept-Workflow(gematik)/1.2.0/e72fee9b-75c7-4292-8ffc-cd81c151df8f.xml]: Might be better to format our examples in a readable manner

**Review by dennishahn — COMMENTED** [review]
What about naming the examples by the profile and version so that one could understand by looking at the filename what to expect?

**Review by bneidhold — APPROVED** [review]

File: Standalone-Examples/E-Rezept-Workflow(gematik)/1.2.0/e72fee9b-75c7-4292-8ffc-cd81c151df8f.xml (hunk 1 above)
Comment (florianschoffke) [reply Standalone-Examples/E-Rezept-Workflow(gematik)/1.2.0/e72fee9b-75c7-4292-8ffc-cd81c151df8f.xml]: done

**Comment by florianschoffke** [comment]
Wird geschlossen, da die Umsetzung der Spezifikation noch erarbeitet wird.

**Pull request closed without merging**